build.js

```diff
@@ -1,17 +1,9 @@
 const { execSync } = require('child_process');
```

Comment:
The `xtc-history` requires the `released optimized` file to exits because of the `include_bytes!` macro, so the build will fail if the user only tries to build the debug mode, can you please update the `xtc-history/xtc-history/src/ic.rs` file to add correct `cfg`s to handle the debug only builds.